hack/gosec.sh

```diff
@@ -9,11 +9,17 @@ cd $KUBEVIRT_DIR/pkg
 
 OUT="${ARTIFACTS}/junit-gosec.xml"
 ARGS="-sort -quiet -out=$OUT -exclude-dir=testutils -fmt=junit-xml ./..."
+DEFAULT_NOSEC="G101"
 if [ ! -z $GOSEC ]; then
     echo "Running subset"
     echo $GOSEC
     ARGS="-include=$GOSEC $ARGS"
 fi
+if [ ! -z $DEFAULT_NOSEC ]; then
+    echo "Exclude rules"
+    echo $DEFAULT_NOSEC
```

Comment:
nit:
Maybe it's better to print them together: `echo "Exclude rules: $DEFAULT_NOSEC"`